Update sos release notes for resolute (#158)
ubuntu/ubuntu-release-notes: docs/26.04/changes-since-previous-interim.md

#### SoS (`sosreport`)

> `sos` was updated to version 4.10.2. Key updates below

Reviewer: ```suggestion
`sos` was updated to version 4.10.2. Key updates include:
```

> - The temporary directory has now been changed from `/tmp` to `/var/tmp`. This follows changed in systemd-tmpfiles and the cleaning of `/var/tmp`, this aligns with other distros.

Reviewer: ```suggestion
- The temporary directory has now been changed from `/tmp` to `/var/tmp`. This follows the change in `systemd-tmpfiles` and the cleaning of `/var/tmp`, and it aligns with other distributions.
```

- Additional plugins include `aws` and `spyre`.

> - The openstack plugins have all been improved to obfuscate passwords more effectively and consistent.

Reviewer: ```suggestion
- The OpenStack plugins have all been improved to obfuscate passwords more effectively and consistently.
```
